app/models/noise.rb

```diff
@@ -73,10 +63,10 @@ class Noise < ActiveRecord::Base
 
   validates_presence_of :provider, :provider_id, :text, :created_at, :user_id
 
-  validates_format_of :avatar_image_url, with: URI.regexp(['http', 'https']), allow_nil: true
+  validates_format_of :avatar_image_url, with: URI.regexp(["http", "https"]), allow_nil: true
```

Comment:
Line is too long. [93/80]<br>Prefer the new style validations `validates :column, format: value` over `validates_format_of`.